sspl_test/run_qa_test.sh

```diff
@@ -152,6 +156,17 @@ restore_cfg_services()
         conf "$global_config" set "server_node>$machine_id>site_id=DC01"
         conf "$global_config" set "server_node>$machine_id>rack_id=RC01"
         conf "$global_config" set "server_node>$machine_id>cluster_id=CC01"
+        if [ "${CVG_NAME}" == "null" ]
+            then
+            if [ "${CVG_INFO}" == "null" ]
+                then
+                `conf $global_config delete "server_node>$machine_id>storage>cvg"`
+            else
+                `conf $global_config delete "server_node>$machine_id>storage>cvg[0]>name"`
+            fi
+        else
+            `conf $global_config set "server_node>$machine_id>storage>cvg[0]>name=${CVG_NAME}"`
+        fi
```

Comment:
Didn't understand this part

Reply:
Here we are restoring the original value of the CVG name which we have updated for sanity @ line no. 246 to 254